examples/run_latex_q_and_a_app.py

```diff
@@ -0,0 +1,45 @@
+import os
+import sys
+sys.path.append(os.path.dirname(os.path.dirname(os.path.realpath(__file__))))
+
+from api import GPT, Example, UIConfig
+from api import demo_web_app
+
+question_prefix = 'Q: '
+question_suffix = "\n"
+answer_prefix = "A: "
+answer_suffix = "\n"
+
+
+# Construct GPT object and show some examples
+gpt = GPT(engine="davinci",
+          temperature=0.5,
+          max_tokens=100,
+          input_prefix = question_prefix,
+          input_suffix = question_suffix,
+          output_prefix = answer_prefix,
+          output_suffix = answer_suffix,
+          append_input_suffix_and_output_prefix_to_query = True,
+          stop = "\nQ:")
+
+gpt.add_example(Example('Two plus two equals four', '2 + 2 = 4', question_prefix, question_suffix, answer_prefix, answer_suffix))
```

Comment:
If you remove the prefixes and suffixes from the `Example` class, then you could simplify to:

```
gpt.add_example(Example('Two plus two equals four', '2 + 2 = 4'))
```

Reply:
Much cleaner - thanks!